File: practice_algos/javascript/algo20.js
// 20. Increment the Seconds - Given an array of numbers arr, add 1 to every
// other element, specifically those whose index is odd(arr[1], arr[3],
// arr[5], etc).Afterward, console.log each array value and return arr

function incrementTheSeconds(arr) {
    for (var i = 0; i < arr.length; i++) {
        if (i % 2 != 0) {
            arr[i]++;
        }
    }
    for (var num in arr) {
        console.log(arr[num]);
    }
    return arr
}

var testArr = [8, 6, 7, 5, 3, 0, 9]
console.log(incrementTheSeconds(testArr))

File: practice_algos/python/algo20.py
# 20. Increment the Seconds - Given an array of numbers arr, add 1 to
# every other element, specifically those whose index is odd (arr[1],
# arr[3], arr[5], etc).  Afterward, print each array value and
# return arr

def increment_the_seconds(arr):
    for i in range(len(arr)):
        if i % 2 != 0:
            arr[i] += 1
        print(arr[i])
    return arr

increment_the_seconds([1, 1, 1, 1, 1, 1, 1, 1])